pql/pqlpeg_test.go

```diff
@@ -333,6 +333,12 @@ func TestPEGErrors(t *testing.T) {
 		{
 			name:  "RangeTimeOneStamp",
 			input: "Row(a=4, 2010-07-04T00:00)"},
+        {
```

Comment:
I think there are some spacing issues here. You'll need to make sure that your editor is configured to run `gofmt` on save. That will make sure that tabs are used in the correct places and all the formatting is correct.